random/module.py

```diff
@@ -35,6 +35,12 @@ async def pick(self, ctx, first: str, second: str, *args):
                 args = args[i + 1:]
                 break
 
+        if len(args) < 2:
+            return await ctx.reply(_(
+                ctx,
+                "You asked a question, but did not add enough options."
+                ))
```

Comment:
Didi you run this through `black`? I believe this is not how it would format the code. We don'ŧ have CI enabled on this repository yet, so it won't show up anywhere.

Reply:
No indeed, I've fixed it and squashed the new commit

ps: There are many lines that needs formatting by Black, it may need another commit IMHO

Reply:
We know. We just haven't found the time to fix it all :)